### PR TITLE
Add configuration required by functional tests

### PR DESCRIPTION
**Pull request by lgonczar**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RPE-302

### Change description ###

Add configuration required by functional tests

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by rkondratowicz — DISMISSED** [review]

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -8,10 +8,34 @@ def component = "consumer"
 
 def channel = '#rpe-build-notices'
 
+List<LinkedHashMap<String, Object>> secrets = [
```
Comment (nitinprabhu) [inline Jenkinsfile_CNP:11]: Could we please test this on CNP branch first before we merge this to master as it will go all the way through prod?

Comment (lgonczar) [reply Jenkinsfile_CNP:11]: Yes, I'll run it from cnp on Sandbox branch before merging. I'll also create keys in Azure Vault.

Comment (nitinprabhu) [reply Jenkinsfile_CNP:11]: Thanks once done will approve this PR

**Comment by lgonczar** [comment]
Here's the successful test job: https://sandbox-build.platform.hmcts.net/job/HMCTS/job/send-letter-consumer-service/job/cnp/29/

I didn't have to add keys to Azure Vault - that happens automatically now

**Review by nitinprabhu — APPROVED** [review]
🏆

**Pull request merged**